lib/api/model/model.dart

```diff
@@ -576,7 +576,10 @@ class StreamMessage extends Message {
   @JsonKey(includeToJson: true)
   String get type => 'stream';
 
-  final String displayRecipient;
+  // This is not nullable API-wise, but if the message move across streams we
+  // might set [displayRecipient] to null to invalidate it.
```

Comment:
Not sure if this comment warrants dart doc given how it is mainly explaining the motivation for the `@JsonKey` here.

Reply:
Yeah, I think a plain implementation comment (like here) is fine.